lib/mongo/server/connection_pool.rb

```diff
@@ -493,14 +507,44 @@ def close_idle_sockets
         end
       end
 
-      # Creates up to the min size connections.
+      # Create and add connections to the pool until the
+      # pool size is at least min_size.
       #
-      # Used by the spec test runner.
+      # Used by the spec test runner and the pool populator background thread.
```

Comment:
The spec runner used this method because we didn't have the background thread. Now that the background thread exists, the spec runner should not be calling this method at all and should instead use the pool normally.